site/pages/resources/index.tsx

```diff
@@ -6,10 +6,31 @@ import { GetStaticPropsContext } from "next";
 export async function getStaticProps(ctx: GetStaticPropsContext) {
   try {
     const { locale } = ctx;
-    const documents = await fetchCMSContent("allDocuments", { locale });
-    const featuredArticles = await fetchCMSContent("allFeaturedPosts", {
-      locale,
-    });
+
+    const [
+      enDocuments,
+      enFeaturedArticles,
+      localeDocuments,
+      localeFeaturedArticles,
+    ] = await Promise.all([
+      fetchCMSContent("allDocuments", { locale: "en" }),
+      fetchCMSContent("allFeaturedPosts", { locale: "en" }),
+      fetchCMSContent("allDocuments", { locale }),
+      fetchCMSContent("allFeaturedPosts", { locale }),
+    ]);
+
+    let documents, featuredArticles;
+    // if en has more documents or articles than locale, then show locale documents and articles plus en documents and articles
+    if (
+      localeDocuments.length < enDocuments.length ||
+      localeFeaturedArticles.length < enFeaturedArticles.length
+    ) {
+      documents = [...localeDocuments, ...enDocuments];
+      featuredArticles = [...localeFeaturedArticles, ...enFeaturedArticles];
```

Comment:
Does this break the sorting? Because the localized articles won't always be first.

Unless we are doing more client side sorting that I'm forgetting about, I believe the GROQ query should sort for us.

We just need to update the GROQ query to match this logic:

```
fetch posts where:
  post.locale === `undefined`
  OR post.locale === 'en'
  OR post.locale === $LOCALE
```